src/content/docs/dropins/payment-services/containers/credit-card.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: CreditCard container
+description: Configure the CreditCard container to manage and display the available credit card fields during a checkout.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+The `CreditCard` container allows a shopper to place an order with credit card details during the checkout process. You can configure it to manage and display the available credit card fields for a payment method.
+
+## CreditCard configurations
+
+The `CreditCard` container provides the following configuration options:
+
+<OptionsTable
+  compact
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['apiUrl', 'string', 'Yes', 'The URL to the Adobe Commerce GraphQL endpoint, such as "https://example.com/graphql".'],
+    ['getCartId', 'function', 'Yes', 'Should return a promise that resolves to the shopper`s cart ID.'],
+    ['getCustomerToken', 'function', 'No', 'The `CreditCard` container may send GraphQL requests on behalf of the shopper. This requires GraphQL authorization, which can be performed using authorization tokens or session cookies. For token-based authorization, the "getCustomerToken" function that should return a customer token as a string, or null for guest checkouts.'],
```

Comment:
Nested markdown is not supported, it seems.

![image](https://github.com/user-attachments/assets/3ed6f3d0-4073-494b-995d-7eec37ac5038)

Can we style this in some other way? If not, let's remove the backticks or use quotes.

Reply:
Also, we should mention what we should pass in here for session-based authorization. We could go for something like this:

> [...] which can be performed using authorization tokens or session cookies. The "getCustomerToken" function should not be provided for session-based authorization. For token-based authorization, on the other hand, the "getCustomerToken" function is required and should return a customer token as a string or null for guest checkouts.